rest/src/db/dbFormattingRules.js

```diff
@@ -40,9 +40,10 @@ module.exports = {
 	[ModelType.objectId]: value => (undefined === value ? '' : value.toHexString().toUpperCase()),
 	[ModelType.statusCode]: value => status.toString(value >>> 0),
 	[ModelType.string]: value => value.toString(),
-	[ModelType.uint]: value => convert.int32ToUint32(value),
+	[ModelType.uint8]: value => convert.int32ToUint32(value),
```

Comment:
why use `int32ToUint32`? in the `ModelType.uint16` we are using different a different approach

Reply:
Both `uint8` and `uint16` maximum values fit inside positive part of `int32`. So we can use `int32ToUint32`. Working on `uint16` case.

Reply:
`int32ToUint32` in fact is not needed because of reason above. Both `uint8` and `uint16` fit inside `int32` positive part. Database will return them in the appropiate format.